-------------------------------------------------------------------------------
Test set: br.com.cotiinformatica.ProjetoSpringApi01ApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 19.404 s <<< FAILURE! - in br.com.cotiinformatica.ProjetoSpringApi01ApplicationTests
contextLoads  Time elapsed: 0.003 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'fornecedoresController': Unsatisfied dependency expressed through field 'fornecedorRepository'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'IFornecedorRepository' defined in br.com.cotiinformatica.interfaces.IFornecedorRepository defined in @EnableJpaRepositories declared on JpaConfiguration: Cannot create inner bean '(inner bean)#195113de' of type [org.springframework.orm.jpa.SharedEntityManagerCreator] while setting bean property 'entityManager'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name '(inner bean)#195113de': Cannot resolve reference to bean 'entityManagerFactory' while setting constructor argument; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [br/com/cotiinformatica/config/JpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on br.com.cotiinformatica.entities.Produto.fornecedor references an unknown entity: br.com.cotiinformatica.entities.Fornecedor
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'IFornecedorRepository' defined in br.com.cotiinformatica.interfaces.IFornecedorRepository defined in @EnableJpaRepositories declared on JpaConfiguration: Cannot create inner bean '(inner bean)#195113de' of type [org.springframework.orm.jpa.SharedEntityManagerCreator] while setting bean property 'entityManager'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name '(inner bean)#195113de': Cannot resolve reference to bean 'entityManagerFactory' while setting constructor argument; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [br/com/cotiinformatica/config/JpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on br.com.cotiinformatica.entities.Produto.fornecedor references an unknown entity: br.com.cotiinformatica.entities.Fornecedor
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name '(inner bean)#195113de': Cannot resolve reference to bean 'entityManagerFactory' while setting constructor argument; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [br/com/cotiinformatica/config/JpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on br.com.cotiinformatica.entities.Produto.fornecedor references an unknown entity: br.com.cotiinformatica.entities.Fornecedor
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [br/com/cotiinformatica/config/JpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on br.com.cotiinformatica.entities.Produto.fornecedor references an unknown entity: br.com.cotiinformatica.entities.Fornecedor
Caused by: org.hibernate.AnnotationException: @OneToOne or @ManyToOne on br.com.cotiinformatica.entities.Produto.fornecedor references an unknown entity: br.com.cotiinformatica.entities.Fornecedor

